src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+interface Props {
+  movies: Movie[];
+}
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map((movie) => (
+      <MovieCard
+        movie={movie}
+        key={movie.title}
```

Comment:
for key we have property  movie.imdbId,

Reply:
Agree, id is `identifier` and thus, it's unique, title's not